akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -940,19 +980,19 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
       context.system.deadLetters // not used
```

Comment:
The comment quoted below (lines 791-798) is now inaccurate and can be re-worded 💯 
```
 * Current protocol for replicating the deltas does not support causal consistency.
  * It is only eventually consistent. This means that if elements 'c' and 'd' are
  * added in two separate `Update` operations these deltas may occasionally be propagated
  * to nodes in different order than the causal order of the updates. For this example it
  * can result in that set {'a', 'b', 'd'} can be seen before element 'c' is seen. Eventually
  * it will be {'a', 'b', 'c', 'd'}. If causal consistency is needed the delta propagation
  * should be disabled with configuration property
  * `akka.cluster.distributed-data.delta-crdt.enabled=off`.
```